### ISSUE TITLE
Can't install the VS plugin from within the Xenko launcher

### ISSUE THREAD

**Issue by z16** [SiliconStudio/xenko#364]
When I try to install the VS plugin from within the launcher, it first downloads it (its status bar changes to "Downloading..."), then installs it (bar changes to "Installing..."), then it's seemingly done. But instead of "Plugin installed" or whatever the text is, the button remains clickable and still says "Install". When you click it, it it changes to "Deleting Visual Studio plugin..." and remains there indefinitely. At no point during this process was the plugin actually installed (I opened Visual Studio to confirm, it wasn't there).

I first had this issue about a week ago, before 1.6 was released. When the Xenko 1.6 upgrade came out, the text changed to "Update". I clicked that, and it downloaded and installed the plugin successfully (for the first time it actually said "Visual Studio plugin installed", and the Xenko menu appeared in Visual Studio).

I have since uninstalled Xenko again, because I was experimenting [how to get the launcher to work](https://github.com/SiliconStudio/xenko/issues/363). After that I had the same issues as before.

Long story short, the updating mechanism was able to install the plugin correctly, but the regular installation mechanism does not seem to work.

**Comment by z16** [comment]
Has been solved by a recent launcher update.
